### PR TITLE
Add arc icons

### LINKED ISSUE #392: Arc Icon Theme
Add please Arc icon theme
https://github.com/horst3180/arc-icon-theme

### PR DESCRIPTION
**Pull request by danstewart**
Not 100% sure on using the current path but using something like /tmp caused some headaches when trying to cleanup.

Let me know if anyone has any better suggestions.

Closes #392.

### CODE REVIEW

**Comment by danstewart** [comment]
Also - Currently the theme doesn't have any application icons but you can set a fallback - if anyone knows how to see what theme is currently in use I can add something to set the fall back to the current theme.

File: plugins/arcicons.plugin/uninstall.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+if [[ -d /usr/share/icons/Arc ]]; then
+		# Can't delete straight from /usr/share/icons/
```
Comment (satya164) [inline plugins/arcicons.plugin/uninstall.sh]: why not?

File: plugins/arcicons.plugin/install.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+url=$(curl -s https://api.github.com/repos/horst3180/arc-icon-theme/releases | grep 'tarball_url' | head -1 | cut -d\" -f4)
+curl -sL $url > arcicons.tar.gz
```
Comment (satya164) [inline plugins/arcicons.plugin/install.sh]: Can you download it to the cache directory (and reuse the cache when user tries to install next time)? https://github.com/folkswithhats/fedy/blob/master/plugins/sublimetext.plugin/install.sh#L3

File: plugins/arcicons.plugin/uninstall.sh (hunk 1 above)
Comment (danstewart) [inline plugins/arcicons.plugin/uninstall.sh]: "The plugin **Arc Icons** is trying to run the command
`rm -f /usr/share/icons/Arc`,
which might **delete important files**.
Continue anyways?"

I hit yes and get a failed notification - running in the terminal has no error messages.

**Comment by danstewart** [comment]
I've added those changes btw.
Any ideas for deleting from /usr/share/icons/?

**Comment by bddenhartog** [comment]
> Any ideas for deleting from /usr/share/icons/?

I just merged your branch in with `master` (locally, for testing), and changed your uninstall script to use the line `rm -rf /usr/share/icons/Arc`. I then moved the `arcicons.plugin` directory into `/usr/share/fedy/plugins`, and started the Fedy application. I can successfully install and uninstall the plugin without any errors or additional prompts.

Can you test this locally (you won't need to do any merging - just modify your file and move the plugin dir into `/usr/share/fedy/plugins`) and let me know what issue you're running into?

**Comment by danstewart** [comment]
@bddenhartog Tried again (now on F25) and I get this popup, I say yes and then I get the failed notification:
https://imgur.com/a/CXfdA

I tried both `rm -rf /usr/share/icons/Arc` and `rm -rf /usr/share/icons/Arc/`

**Comment by danstewart** [comment]
I'll close this and send over another PR using the copr when I have time.

**Comment by bddenhartog** [comment]
@danstewart I'm not sure why you would be getting that popup in the first place; I don't receive it at all. I think using the copr is a good idea, as long as that repo is maintained!

**Pull request closed without merging**